### ISSUE TITLE
"Missing archive file please-vote-on-jed.zip" Odd error

### ISSUE THREAD

**Issue by fstoltz** [AlexRed/ozio#4]
Would you mind going onto **www.kadagaya.org**?

It seems to be an error originating from oziogallery, are you familiar with this?

Many thanks for any help.

**Comment by AlexRed** [comment]
Hello fstoltz,
no, I'm not familiar with this.
Try to reinstall the last Ozio Gallery version.

**Comment by AlexRed** [comment]
and update your Joomla version

**Comment by fstoltz** [comment]
Thanks for such a swift reply.

I've updated Joomla now, and the site seems to be working now. Though when going to the actual Ozio Gallery page I'm getting some different errors now.

This page: http://kadagaya.org/index.php/en/photo-gallery



You know what's causing this?

**Comment by AlexRed** [comment]
Do you create the Google credential API ?
Do you insert it in the Ozio Gallery component page ?
See instructions: https://www.opensourcesolutions.es/en/ext/ozio-gallery.html#Instructions

**Comment by fstoltz** [comment]
Yes, now Ive both created the credential API and inserted into the component page. But im still getting the same error. Odd. I followed the .gif with how to do all the things and everything seems like its properly setup.

Any ideas?

**Comment by AlexRed** [comment]
post the screenshot of you Ozio Gallery configuration and of your Google API credential configuration.

**Comment by fstoltz** [comment]
![aipipic](https://cloud.githubusercontent.com/assets/22579564/25278474/a8b447a2-2668-11e7-9874-615205e0bafe.PNG)
__________________________________________________
![apipic](https://cloud.githubusercontent.com/assets/22579564/25278475/a8d0dd86-2668-11e7-94c4-73d46bafaade.PNG)
____________________________________________________
![picc](https://cloud.githubusercontent.com/assets/22579564/25278476/a8da90ba-2668-11e7-9e72-8a119f3c9294.PNG)

**Comment by AlexRed** [comment]
seems all ok.
Do you try the same "authorized" in other site and delete it ?

Try to delete and re-enter the "authorized" in your credential in the Ozio Component page.

**Comment by fstoltz** [comment]
Nope, it's a "raw" authorized. Haven't used it on any other sites, it's freshly created.

Tried it, though no luck, still showing the same error. 

You think it might have to do with any of the specific settings for my Ozio Gallery item?

**Comment by AlexRed** [comment]
I think no, but try to create a new Ozio Gallery item menu with skin nano and the default setting and default Google ID. Only create a new menu item and save it.

**Comment by fstoltz** [comment]
I created a new menu item with Skin Nano with default settings but still received the same error I believe.

What could it be..? Ozio Gallery is updated to latest version 4.8.1

Maybe I'll try setting it up with a different google account to see if it works. Might be an issue with the Google Photos album settings? I haven't changed anything in there for quite a while.

**Comment by AlexRed** [comment]
Do you use Artio SEF or sh404sef or JCH Optimize  ?

**Comment by fstoltz** [comment]
Not that I know of, don't recognize any of them. (by the way, I just updated Joomla to 3.7.0, same error still)

**Comment by fstoltz** [comment]
I tried plenty different things, different PHP versions, different browsers, messing about with the google API and where the albums are stored, sharing settings etc. 

But in the end the gallery started working again after I went into Extensions > Manage > Manage > Disabled & Enabled all the Ozio ext. > Refreshed the cache for all ozio ext. And then it started working again!

Thanks a lot for fast replies & helping to troubleshoot though.
